utils/vvmtools.py

```diff
@@ -82,6 +82,22 @@ def cal_pblh_maxgrad(self,t, func_config):
         max_idx = np.nanargmax(del_th,axis=0)
         return self.DIM['zc'][max_idx]
 
+    def cal_pblh_ens(self, t, func_config):
+        ens=self.cal_enstrophy(t, func_config)
+        if np.max(ens)>=func_config['ens_threshold']:
+            idx=self.nz-np.where(ens[::-1]>=func_config['ens_threshold'])[0][0]
+            return self.DIM['zc'][idx]
+        else:
+            return np.nan
+
+    def cal_pblh_tke(self, t, func_config):
+        tke=self.cal_TKE(t, func_config)
+        if np.max(tke)>=func_config['tke_threshold']:
```

Comment:
i think the threshold in func_config does not distinguish take or ens.
can use same key name

Reply:
Sounds good to me.
I will close this request and update later.